modules/odc-using-web-terminal.adoc

```diff
@@ -29,3 +29,8 @@ After the web terminal is initialized, you can use the preinstalled CLI tools li
 * The web terminal remains open until you close it or you close the browser window or tab.
 ====
 
+== Web terminal and network policies
```

Comment:
This is missing an `id` for this heading in the form of `[id="web-terminal-and-network-policies_{context}"]`